### PR TITLE
Updated versions for storage Feb release

### PR DESCRIPTION
**Pull request by gapra-msft**

### CODE REVIEW

File: sdk/storage/azure-storage-blob-batch/README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ definition, such as text or binary data.
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-blob-batch</artifactId>
-  <version>12.8.0</version>
+  <version>12.9.0-beta.1</version>
```
Comment (g2vinay) [inline sdk/storage/azure-storage-blob-batch/README.md:25]: we generally, keep README at stable versions, so existing and new customers try the stable release.

File: sdk/storage/azure-storage-blob-cryptography/README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ This package supports client side encryption for blob storage.
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-blob-cryptography</artifactId>
-  <version>12.10.0</version>
+  <version>12.11.0-beta.1</version>
```
Comment (g2vinay) [inline sdk/storage/azure-storage-blob-cryptography/README.md:24]: we generally, keep README at stable versions, so existing and new customers try the stable release.

**Review by g2vinay — COMMENTED** [review]
we should keep the READMEs to point at stable versions.
So, that existing and new customers try the latest stable with confidence and not get nervous to see the beta as the recommended package to try.

File: sdk/storage/azure-storage-blob-batch/README.md (hunk 1 above)
Comment (gapra-msft) [reply sdk/storage/azure-storage-blob-batch/README.md]: Fixed

**Review by alzimmermsft — APPROVED** [review]

**Review by g2vinay — APPROVED** [review]

**Pull request merged**